### ISSUE TITLE
ERR_ABORTED webcomponents-loader.js at localhost

### ISSUE THREAD

**Issue by AndyOGo** [axa-ch/patterns-library#278]
> GET http://localhost:3000/app/webcomponents-loader.js net::ERR_ABORTED

<img width="1439" alt="screen shot 2018-02-26 at 17 45 26" src="https://user-images.githubusercontent.com/914443/36683042-dbe64086-1b1c-11e8-81ce-983ba95f2317.png">

**Comment by AndyOGo** [comment]
`webcomponent-*` polyfill files are missing in `./tmp`:

<img width="416" alt="screen shot 2018-02-27 at 20 59 37" src="https://user-images.githubusercontent.com/914443/36751672-18e707d8-1c01-11e8-8221-278360202d72.png">

**Comment by AndyOGo** [comment]
Those lines should do the trick, but somehow doesn't...
https://github.com/axa-ch/patterns-library/blob/master/stack/tasks/copy-polyfills.js#L7-L21

**Comment by AndyOGo** [comment]
Okay I tracked it down 🎉 
**TLDR:** `npm run serve` doesn't call `build-js-dev` script, which would call `copy-polyfills`.
Instead it just calls `watch-js` script, which doesn't call the webcomponent polyfill stuff...
